annotations/utils.py

```diff
@@ -290,12 +292,15 @@ def bind_annotations_to_xml(source):
 
             words = annotation.words.all()
             for w in words:
-                xml_w = tree.xpath('//w[@id="{}"]'.format(w.xml_id))
-                if len(xml_w) != 1:
-                    failed_lookups.append(annotation)
-                    continue
-
-                xml_w = xml_w[0]
+                words_by_xml_id[w.xml_id] = dict(annotation=annotation, tense=tense_label, color=tense_color, found=False)
+
+        for xml_w in tree.xpath('//w'):
+            word = words_by_xml_id.get(xml_w.get('id'))
+            if word:
+                word['found'] = True
```

Comment:
Instead of setting this variable to True, you could have also considered dropping this word from the dictionary.